src/module/actor/pokemon/document.js

```diff
@@ -354,27 +354,55 @@ class PTUPokemonActor extends PTUActor {
 
     // TODO: Implement rules for capability changing items
     _calcCapabilities() {
-        const capabilities = duplicate(this.species?.system?.capabilities ?? {});
-        if (!capabilities) return {};
+        const numericNonMovementCapabilities = CONFIG.PTU.Capabilities.numericNonMovement
+        const stringArrayCapabilities = CONFIG.PTU.Capabilities.stringArray
+
+        const speciesCapabilities = duplicate(this.species?.system?.capabilities ?? {});
+        const finalCapabilities = {}
+        if (!speciesCapabilities) return {};
```

Comment:
Don't initialize just to return the same initialization after. Swap these lines around.